doc/en/user/source/tutorials/imagemosaic_footprint/imagemosaic_footprint.rst

```diff
@@ -202,3 +202,27 @@ Step 4: Check the result
 Go to :menuselection:`LayerPreview --> rastermask --> OpenLayers`. The result should be changed now.
 
 .. figure:: img/footprint_transparent.png
+
+
+Multilevel Geometry Masking
+---------------------------
+
+From 2.14.x version, GeoServer is able to support also multilevel overviews geometries (A geometry footprint for each overview, being stored on a separate sidecar file).
+
+
+A **footprints.properties** file that would exploit multiple WKB sidecar files would be as follows::
+
+	footprint_source=multisidecar
+	footprintLoaderSPI=org.geotools.coverage.grid.io.footprint.WKBLoaderSPI
+	overviewsFootprintLoaderSPI=org.geotools.coverage.grid.io.footprint.WKBLoaderSPI
+	overviewsRoiInRasterSpace=True
+	overviewsSuffixFormat=_%d
+	
+Notes:
+	
+	* *footprintLoaderSPI*: Contains the fully qualified name of the SPI implementation for main footprint loading (Mandatory property) (WKT loading is supported by org.geotools.coverage.grid.io.footprint.WKTLoaderSpi);
```

Comment:
I would add a list of the values to be used. Think the original overview provider was able to detect wkt or wkb on its own, without an indication?